src/main/groovy/com/bmuschko/gradle/docker/DockerSpringBootApplication.groovy

```diff
@@ -55,9 +55,28 @@ class DockerSpringBootApplication {
      * The tag used for the Docker image.
      * <p>
      * Defaults to {@code <project.group>/<applicationName>:<project.version>}.
+     * @deprecated use {@link #tags} - will be removed in 6.0.0
      */
+    @Deprecated
     final Property<String> tag
 
+    /**
+     * The tags used for the Docker image.
+     * <p>
+     * Defaults to {@code [<project.group>/<applicationName>:<project.version>]}.
+     */
+    final ListProperty<String> tags
+
+    /**
+     * Tags that will be used for building the Docker image, but will not be pushed to the repo.
+     * In order for these tags to be built, they should be specified in the {@link #tags} property.
+     * e.g. for rapid local development, "busybox:latest" would be useful, but becomes confusing when
+     * pushed to a remote repo
+     * <p>
+     * Defaults to {@code []}
+     */
+    final ListProperty<String> localOnlyTags
```

Comment:
Do we really need the `localOnlyTags`? Just trying to think what the usecase is here.

Reply:
So I was thinking of my team's specific usecase, where we build "<application>:latest" and reference it in all of our kubernetes manifests so that its fairly easy to do local development.

However, we don't want those "latest" tags to ever go to our docker repo, we just want them to stay local. This was just the first way I thought of doing it, I'm open to other suggestions.

Reply:
Ok I can see that being useful and handy for others. Can we change the name though to something more descriptive? Maybe `skipPushTags` or `skipPushOnTags` or if there is something better.

Reply:
Those names are definitely better than `localOnlyTags`.

How about `noPushTags`?

Reply:
Hmmm ... I'm on the line for `noPushTags` and `skipPushTags`. I leave the ball in your court ;)